src/web/Html.tsx

```diff
@@ -77,23 +98,31 @@ export const Html = React.forwardRef(
       fullscreen,
       portal,
       scaleFactor,
+      sprite = false,
+      transform = false,
       zIndexRange = [16777271, 0],
       calculatePosition = defaultCalculatePosition,
       ...props
     }: HtmlProps,
     ref: React.Ref<HTMLDivElement>
   ) => {
     const { gl, scene, camera, size } = useThree()
-    const [el] = React.useState(() => document.createElement('div'))
+    const el = React.useMemo(() => document.createElement('div'), [])
     const group = React.useRef<Group>(null)
     const old = React.useRef([0, 0])
-    const target = portal?.current ?? gl.domElement?.parentNode
+    const transformOuterRef = React.useRef() as React.MutableRefObject<HTMLDivElement>
```

Comment:
As soon as I do this change, I receive a typescript error saying that transformOuterRef.current can be undefined; I think typescript infers HTMLDivElement | undefined because I'm not passing any initial value; but I'm not sure here, if you have suggestions on how to avoid this problem I will edit this

Reply:
yeah so the return type here is `{ current: HTMLDivElement | undefined }`. you'd just have to check that `transformOuterRef.current` is defined before using it, which imo we should be doing in this case as it's not always guaranteed that the ref will be used in the component since it's not always attached.

Reply:
if it's not a problem I will write it as React.useRef<HTMLDivElement>(null), because with undefined as a possible type I get a ref error; I think it's okay because you already write React.useRef<Group>(null)